src/csharp/Microsoft.Spark/SparkContext.cs

```diff
@@ -312,6 +313,13 @@ public Broadcast<T> Broadcast<T>(T value)
             return new Broadcast<T>(this, value);
         }
 
+        /// <summary>
+        /// A default Hadoop Configuration for the Hadoop code (e.g. file systems) that we reuse.
+        /// </summary>
+        /// <returns>The Hadoop Configuration.</returns>
+        public Configuration HadoopConfiguration() =>
```

Comment:
@imback82 Not sure if this should be a function or a property.

Reply:
Since this is `def` on the Scala side, this will be a method on C# side. So the current approach is good.